data/heliarch intro.txt

```diff
@@ -0,0 +1,1570 @@
+	landing
+	source
+		government "Heliarch"
+		not planet "Ring of Wisdom"
+	destination "Ring of Wisdom"
+	to offer
+		has "event: heliarch recon break"
+		not "Resistance: Questions: offered"
+	on offer
+		conversation
+			`Some Heliarchs approach you as you land. You think you can recognize the Saryd of the group as one of those who asked you to land on all the Quarg-owned worlds some time ago.`
+			`	"The honorable Captain, we once again are pleased to greet," he says.`
+			`	"Another fortuitous meeting, we hope this will be."`
+			`	"In mind, a new strategy we have. Your assistance, we once again require."`
+			choice
+				`	"What do you need me to do?"`
+				`	"I'm sorry, but I'm not willing to risk my ship any longer to help you spy on the Quarg."`
+					decline
+			`	"If too complex and protected, the Quarg's rings are, perhaps something new, from their ships we might learn."`
+			`	"Scan them during our war, we of course did, but much wiser and more sofisticated, we and our devices have become."`
+			`	"To use one of our own scanners, we cannot ask of thee. Much risk, it would pose. Alas, if to <destination>, a human Outfit Scanner you bring, alter it, we could."`
+			`	You agree to head there once you have equipped an Outfit Scanner, and they thank you for your willingness to help.`
+				accept
+	on complete
+		require "Outfit Scanner"
+
+
+
+mission "Heliarch Recon 2-B"
+	name "Silent Scan"
+	description "Now that the Heliarchs have altered your Outfit Scanner, head to <waypoints> and scan some Quarg ships there, then report back to <destination>."
+	landing
+	source "Ring of Wisdom"
+	destination "Ring of Wisdom"
+	waypoint "Enif"
+	to offer
+		has "Heliarch Recon 2-A: done"
+		not "Resistance: Questions: offered"
+	on offer
+		conversation
+			`While you don't find any of the Heliarchs that spoke to you here, one of their groups is there with a team of engineers once you land.`
+			`	They remove the Outfit Scanner from your ship, and take it to some restricted area. After about an hour, they come back, and while you can't spot any change done to the oufit, they reinstall it in your ship.`
+			`	They say they will be waiting for your return, and wish you good luck.`
+				accept
+	npc "scan outfits"
+		personality staying uninterested target
+		government "Quarg"
+		system "Enif"
+		fleet
+			names "quarg"
+			variant
+				"Quarg Skylark"
+				"Quarg Wardragon"
+	on complete
+		payment 200000
+		conversation
+			`Once again the engineers remove your Outfit Scanner, taking a copy of the scan logs with them, and handing another copy to the Heliarchs, who hand you <payment>.`
+			`	They return the outfit to your ship just like before, you still not being able to tell what they did to it.`
+			`	"Grateful to you, we are, Captain <last>," the Saryd says. "Remember your assistance, we will."`
+
+
+
```

Comment:
Maybe you could add some flavor text here suggesting the player go to the Quarg worlds to see what was going on when they all heard your scanner's whirring sound? It seems like you might leave the player semi-stranded here not knowing what to do next.